app/controllers/api/v1/discussions_controller.rb

```diff
@@ -0,0 +1,17 @@
+class API::V1::DiscussionsController < API::V1::ApplicationController
+
+  before_action :load_discussion, only: [:attendees]
+
+  def attendees
```

Comment:
Shouldn't this action be part of `DiscussionsUsersController` ? Since you have made one..